testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/api/expression/ExpressionSubstitutionInContainerTestCase.java

```diff
@@ -208,44 +205,44 @@ public void testMultipleLevelExpression() {
         Utils.setProperty(EXPRESSION_PROP_NAME, EXPRESSION_PROP_VALUE, managementClient.getControllerClient());
         Utils.setProperty(INNER_PROP_NAME, "${" + EXPRESSION_PROP_NAME + ":" + INNER_PROP_DEFAULT_VALUE + "}", managementClient.getControllerClient());
         Utils.setProperty(PROP_NAME, "${" + INNER_PROP_NAME + ":" + PROP_DEFAULT_VALUE + "}", managementClient.getControllerClient());
-                
+
         try {
             // evaluation the inner prop name in addition
             String result = bean.getJBossProperty(INNER_PROP_NAME);
             log.infof("expressionEvaluation: JBoss property %s was resolved to %s", INNER_PROP_NAME, result);
             Assert.assertEquals("jboss property " + INNER_PROP_NAME + " substitution evaluation expected", EXPRESSION_PROP_VALUE, result);
-    
+
             result = bean.getSystemProperty(INNER_PROP_NAME);
             log.infof("expressionEvaluation: System property %s has value %s", INNER_PROP_NAME, result);
             Assert.assertEquals("system property " + INNER_PROP_NAME + " from substitued jboss property", EXPRESSION_PROP_VALUE, result);
-            
+
             // then evaluation of the rest
             expresionEvaluation();
         } finally {
             // removing tested properties
             Utils.removeProperty(EXPRESSION_PROP_NAME, managementClient.getControllerClient());
-            Utils.removeProperty(PROP_NAME, managementClient.getControllerClient()); 
+            Utils.removeProperty(PROP_NAME, managementClient.getControllerClient());
             Utils.removeProperty(INNER_PROP_NAME, managementClient.getControllerClient());
         }
     }
-    
+
     /**
      *  <system-properties>
      *    <property name="first.defined.value" value="expression.value"/>
      *    <property name="qa.test.property" value="${qa.test.exp:defaultValue}"/>
      * </system-properties>
-     * 
+     *
      * Write attribute set:
      * <property name="qa.test.exp" value="expression.value"/>
      */
-    @Ignore("AS7-6431")  // for this test works there will be :reload after redefinition
     @Test
+    @Ignore("AS7-6431")
```

Comment:
Feel free to comment out (don't delete) the tests that are ignored due to AS7-6431. I've rejected that JIRA.